### DISCUSSION TITLE
Where can i download lastest prebuilt wine for box86

### DISCUSSION THREAD

**Discussion by xDoge26** [ptitSeb/box86#715] [Q&A] [answered]
i want to use wine staging with box86

**Comment by WheezyE** [comment, marked as answer]
You might try these install instructions (but replace “wbranch=devel” with
“wbranch=staging”)
https://github.com/ptitSeb/box86/blob/master/docs/X86WINE.md#installing-wine-for-box86-on-raspberry-pi-os-from-winehq-deb-files
